packages/core/lib/commands/test/run.js

```diff
@@ -107,6 +107,27 @@ module.exports = async function (options) {
     }
   };
 
+  function sanitizeGanacheOptions(ganacheOptions) {
```

Comment:
One other thing to consider is pulling this out of the main body of this function. Style-wise I dislike having functions defined like this in the middle of another function as it disturbs the narrative and makes it harder to understand the code flow.

Reply:
Sure, @eggplantzzz . I will do that. Thanks.